src/comm.F

```diff
@@ -11158,6 +8397,9 @@ subroutine SetMsgParams(openacc,gdirect,device,gpudirect)
          logical, intent(inout) :: openacc, gdirect
          logical, optional :: device,gpudirect
 
+      ! Set logical flags if subroutine is called on GPU or CPU resident
+      ! data. The default to for GPU resident data. If the data is GPU
```

Comment:
similar question about the comment `The default to for GPU resident data` here.

Reply:
I corrected the comment.  Please verify that it makes more sense now.

Reply:
Thanks John. It looks good to me now.